Sources/Client/Model/Channel/Channel.swift

```diff
@@ -111,7 +111,8 @@ public final class Channel: Codable {
     public var isUnread: Bool { unreadCount.messages > 0 }
     /// An option to enable ban users.
     public var banEnabling = BanEnabling.disabled
-    var bannedUsers = [User]()
+    /// List of banned users. Should not be edited manually, automatically updated after `banned` or `unbanned` websocket events.
+    public var bannedUsers = [User]()
```

Comment:
I'm not sure it's a good idea to make it public. It feels like we designed something wrong. As I see we only need it only to make unban in `ChannelPresenter` but on the same time we have `Channel.ban()` request where we update `bannedUsers` in completion block. Maybe you can do the same for the unbanning in the Client event handler.